pydicom/dataelem.py

```diff
@@ -87,7 +87,7 @@ def isMultiValue(value):
     """
```

Comment:
Isn't `isMultiValue` going to be removed in v2.0 anyway?

Reply:
Yes, good point, but maybe we can do deprecated in a separate PR - there is already a lot going on here.